### PR TITLE
Feature /SM-1 Implemented the login story

### PR DESCRIPTION
**Pull request by Mihalachirazvan**

### CODE REVIEW

**Comment by MRazvanFIS** [comment]
"Please resolve conflicts!"

File: src/loose/oose/fis/lab/student/manager/Controller.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package loose.oose.fis.lab.student.manager;
```
Comment (MRazvanFIS) [inline src/loose/oose/fis/lab/student/manager/Controller.java]: Remove unused file

File: src/loose/oose/fis/lab/student/manager/Main.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package sample;
```
Comment (MRazvanFIS) [inline src/loose/oose/fis/lab/student/manager/Main.java]: Remove unused file

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            return;
+        }
+
+        if (username.equals("student") && password.equals("student")) {
+            loginMessage.setText("Logged in as student!");
+            return;
+        }
+
+        if (username.equals("teacher") && password.equals("teacher")) {
+            loginMessage.setText("Logged in as teacher!");
```
Comment (MRazvanFIS) [inline src/loose/oose/fis/lab/student/manager/controllers/LoginController.java]: If login as a teacher succeeds, the user should be taken to the view-students scene

File: src/loose/oose/fis/lab/student/manager/Main.java (hunk 2 above)
Comment (Mihalachirazvan) [reply src/loose/oose/fis/lab/student/manager/Main.java]: it should be sample.fxml file

**Pull request merged**